docs/content/manual/pre-release/cluster-restore/node-creation-deletion-with-restore.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Node creation and deletion with restore
+---
+
+Notice that the behaviors will be different if the cluster node roles are different. e.g., A cluster contains 1 dedicated master node + 3 worker node is different from a cluster contains 3 nodes which are both master and worker.
+This test may need to be validated for both kind of cluster.
+
+1. Deploy a 3-worker-node cluster then install Longhorn system.
+2. Deploy some workloads using Longhorn volumes then write some data.
+3. Create a cluster snapshot via Rancher.
+4. Add a new worker node for this cluster. Deploy workloads on this node.
```

Comment:
Do you mean, Deploy workloads attached with new volumes?

Reply:
Yes